ios/RNLinksdk.m-e

```diff
@@ -53,7 +53,7 @@ NSString* const kRNLinkKitDepositSwitchTokenPrefix = @"deposit-switch-";
 RCT_EXPORT_MODULE();
 
 + (NSString*)sdkVersion {
-    return @"7.0.3"; // SDK_VERSION
+return @"7.0.3"; // SDK_VERSION
```

Comment:
What is this -e file?

Reply:
not sure, this just gets updated from the version command https://eng-docs.plaid.com/architecture/mobile-sdks/react-native/releasing.html#preparing-a-new-version-commit